### ISSUE TITLE
Specular Highlights

### ISSUE THREAD

**Issue by toonjoey** [shaddatic/sa2b-render-fix#3]
The grabbable pulleys are supposed to have a specular effect. This isn't rendered in the HD rereleases, resulting in a very flat looking pulley.

Dreamcast:
![image](https://github.com/shaddatic/sa2b-render-fix/assets/19507339/54d28760-d942-4bc1-98e7-c3c7798a46b2)
HD:
![image](https://github.com/shaddatic/sa2b-render-fix/assets/19507339/9a907cb5-abd7-4d46-ab24-cf74c5ab6000)

**Comment by shaddatic** [comment]
This is specular highlighting, and as such I'm going to broaden this issue and link any other specular issues back to this as duplicates. 

Specular is a rendering technique, different from environment maps, that brightens model vertexes facing the light source - in basic terms, the opposite of shading.

This is very likely to be restored one day, but not by me. When it does get done, this mod or not, I'll close this issue. Until then, I'll leave it open for further discussion. Feel free to add other objects missing specular here.

**Comment by LostImpactProd** [comment]
The only other objects I can think off the top of my head that's missing specular highlighting from the DC version are the item balloons and some of the GUN robots (most notably the Beetles).

**Comment by toonjoey** [comment]
I _think_ there's a tiny bit of specular on the rings as well? Going off the video I've been referencing at least.
![image](https://github.com/shaddatic/sa2b-render-fix/assets/19507339/b2a81fbc-0937-41fb-b8ee-f2e643273a2c)

**Comment by TorutheRedFox** [comment]
a lot of object are supposed to have specular but they haven't had it since the GameCube release

**Comment by NikkMann** [comment]
Yeah watching some DC footage looks like many dynamic objects have it. Springs, almost all robots, pulleys, checkpoints, rockets, and it might look like the karts, mechs, and metal bits on the biolizard use it as well.
